sdk/src/types/public/anchor-types.ts

```diff
@@ -32,10 +32,13 @@ const IDL = WhirlpoolIDL as Idl;
 export const WHIRLPOOL_CODER = new BorshAccountsCoder(IDL);
 
 /**
- * Size of the Whirlpool account in bytes.
- * @category Solana Accounts
+ * Get the size of an account owned by the Whirlpool program in bytes.
+ * @param accountName Whirlpool account name
+ * @returns Size in bytes of the account
  */
-export const WHIRLPOOL_ACCOUNT_SIZE = WHIRLPOOL_CODER.size(IDL.accounts![4]);
+export function getAccountSize(accountName: AccountName) {
+  return WHIRLPOOL_CODER.size(IDL.accounts!.find((account) => account.name === accountName)!);
```

Comment:
IDL doesn't contain the calculated account-size and the Coder calculates the total size of each field.
And PositionBundle has 64 bytes reserved space for the future use.

https://github.com/orca-so/whirlpools/blob/main/programs/whirlpool/src/state/position_bundle.rs#L9-L16
https://github.com/orca-so/whirlpools/blob/main/sdk/src/artifacts/whirlpool.json#L1893-L1914

So I think we need to use the following patch. (of course, this patch is not cool 😨)
```
const reserved = (accountName === AccountName.PositionBundle) ? 64 : 0;
return WHIRLPOOL_CODER.size(.....) + reserved;
```